components/data.ts

```diff
@@ -92,4 +92,8 @@ export const MainRoutes = (): Link[] => [
     title: t('about_us'),
     path: 'https://kaiyuanshe.feishu.cn/wiki/wikcn749HAOCD2dwaNq4dOC67db',
   },
+  {
+    title: t('issues'),
+    path: '/issues',
+  },
```

Comment:
```suggestion
  {
    title: t('issues'),
    path: '/project/issues',
  },
```
放在“精彩活动”、“合作伙伴”之间，加个“开源项目”折叠菜单，并放在其下。